Procfile

```diff
@@ -2,4 +2,7 @@
 # If you wish to use a different server then the default, use e.g. `export RAILS_STARTUP='puma -w 3 -p 3000 --preload'`
 rails: [ -n "$RAILS_STARTUP" ] && env PRY_WARNING=1 $RAILS_STARTUP || [ -n "$BIND" ] && bin/rails server -b $BIND || env PRY_WARNING=1 bin/rails server
 # you can use WEBPACK_OPTS to customize webpack server, e.g. 'WEBPACK_OPTS='--https --key /path/to/key --cert /path/to/cert.pem --cacert /path/to/cacert.pem' foreman start '
-webpack: [ -n "$NODE_ENV" ] && ./node_modules/.bin/webpack-dev-server-without-h2 --config config/webpack.config.js $WEBPACK_OPTS || env NODE_ENV=development ./node_modules/.bin/webpack-dev-server-without-h2 --config config/webpack.config.js $WEBPACK_OPTS
+webpack: [ -n "$NODE_ENV" ] && npx webpack --config config/webpack.config.js --watch $WEBPACK_OPTS || env NODE_ENV=development npx webpack --config config/webpack.config.js --watch
+#TODO readme/forklift to change/remove --key to --server-options-key
+# --public -> --client-web-socket-url and see why it doesnt work
+# --https --> --server-type
```

Comment:
This is https://github.com/theforeman/puppet-katello_devel/blob/596745918fa758f209dbffb81d673c6c011186c9/templates/env.erb#L4. It's not exactly clear to me what needs to change. Can you submit a PR?

Reply:
Since we wont be using webpack as a server, we need to remove those.
I'm not sure when will be a good time to remove it in case people (like me) use forklift for older foreman versions.
For now I'm filtering these options out with `FILTERED_WEBPACK_OPTS` but if users somehow use these options in webpack (for example, running `npx webpack --config config/webpack.config.js --https`) webpack will not run.

Reply:
I think that's an acceptable solution.